Resources/Views/Shared/_footer.leaf

```diff
@@ -30,7 +30,7 @@
 
   <div class="row">
     <p class="col-4">
-      Copyright &copy; 2022 SwiftLeeds
+      Copyright &copy; 2021-<script>document.write(new Date().getFullYear())</script> SwiftLeeds
```

Comment:
If preferred, this can be done with a LeafTag.  I already have one written to handle this scenario that I can share?